python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -327,3 +328,15 @@ def load_file(
         enable_native_fallback=enable_native_fallback,
         **kwargs,
     ).output
+
+
+def check_if_connection_exists(conn_id: str) -> bool:
+    """
+    Given an Airflow connection ID, identify if it exists.
+    Return True if it does or raise an AirflowNotFoundException exception if it does not.
+
+    :param conn_id: Airflow connection ID
+    :return bool: If the connection exists, return True
+    """
+    BaseHook.get_connection(conn_id)
```

Comment:
@kaxil don't we need to catch the exception?

Reply:
No, the intention is if the connection doesn't exist, `BaseHook.get_connection(conn_id)` will raise `AirflowNotFoundException` :) 

This PR just moves the method without changing the logic

Reply:
Okay, but we have a similar function somewhere in the code base which wraps it in a try-except.

Reply:
hmm interesting, I couldn't find it in Python SDK